### ISSUE TITLE
Question about the Assignment

### ISSUE THREAD

**Issue by natsuko68** [anyone-can-cook/rclass1#94]
① For question ４, from the small question 2, we need to type "typeof(), length(), and str()".
In that process, do we put the age separately? (eg: typeof(3), typeof(6)) If so, the answer shows all the same. 
Could you please teach me how to do this process?

②The files must be submitted in PDF and RMD.
I am now doing the questions separately on R, and typing the answers in a Word file.
In this case, is it possible to download the Word files in R studios to create PDF and RMD?(I've already complete for making an environment for Knit).

Natsuko Ogura

**Comment by EuphieGe** [comment]
No, we just need to use typeof(), length(), and str() for the age object, like this：
`typeof(age)
length(age)
str(age)`
